diff --git a/node_modules/gpx-builder/dist/builder/BaseBuilder/BaseBuilder.js b/node_modules/gpx-builder/dist/builder/BaseBuilder/BaseBuilder.js
index 61749fb..f84c25c 100644
--- a/node_modules/gpx-builder/dist/builder/BaseBuilder/BaseBuilder.js
+++ b/node_modules/gpx-builder/dist/builder/BaseBuilder/BaseBuilder.js
@@ -95,6 +95,7 @@ class BaseBuilder {
         'creator': 'fabulator:gpx-builder',
         'version': '1.1',
         'xmlns': 'http://www.topografix.com/GPX/1/1',
+        'xmlns:osmand': 'https://osmand.net',
         'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
         'xsi:schemaLocation': this.schemaLocation.join(' '),
         ...this.data.attributes
diff --git a/node_modules/gpx-builder/es/builder/BaseBuilder/BaseBuilder.js b/node_modules/gpx-builder/es/builder/BaseBuilder/BaseBuilder.js
index 46a33e7..a26d263 100644
--- a/node_modules/gpx-builder/es/builder/BaseBuilder/BaseBuilder.js
+++ b/node_modules/gpx-builder/es/builder/BaseBuilder/BaseBuilder.js
@@ -93,6 +93,7 @@ class BaseBuilder {
         'creator': 'fabulator:gpx-builder',
         'version': '1.1',
         'xmlns': 'http://www.topografix.com/GPX/1/1',
+        'xmlns:osmand': 'https://osmand.net',
         'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
         'xsi:schemaLocation': this.schemaLocation.join(' '),
         ...this.data.attributes
diff --git a/node_modules/gpx-builder/src/builder/BaseBuilder/BaseBuilder.ts b/node_modules/gpx-builder/src/builder/BaseBuilder/BaseBuilder.ts
index e45b28f..85aaf5d 100644
--- a/node_modules/gpx-builder/src/builder/BaseBuilder/BaseBuilder.ts
+++ b/node_modules/gpx-builder/src/builder/BaseBuilder/BaseBuilder.ts
@@ -93,6 +93,7 @@ export default class BaseBuilder {
                 'creator': 'fabulator:gpx-builder',
                 'version': '1.1',
                 'xmlns': 'http://www.topografix.com/GPX/1/1',
+                'xmlns:osmand': 'https://osmand.net',
                 'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
                 'xsi:schemaLocation': this.schemaLocation.join(' '),
                 ...this.data.attributes,
